_posts/2020-06-01-Fortran-Newsletter-June-2020.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "Fortran newsletter: June 2020"
+category: newsletter
+author: Jeremie Vandenplas, Brad Richardson, and Milan Curcic
+---
+
+Welcome to the June 2020 edition of the monthly Fortran newsletter.
+The newsletter comes out on the first calendar day of every month
+and details Fortran news from the previous month.
+
+* [fortran-lang.org](#fortran-lang.org)
+* [Fortran Discourse](#fortran-discourse)
+* [Standard Library](#fortran-standard-library)
+* [Package Manager](#package-manager)
+* [Events](#events)
+
+## fortran-lang.org
+
+The Fortran website has been up since mid-April, and we've already got great
+feedback from the community already.
+In the past month we've updated the [Compilers](/compilers) page which is now
+comprehensive and includes all major open source and commercial compilers.
+The [Learn](/learn) page has also seen significant updates—it's been
+reorganized for easier navigation and currently features a quickstart tutorial,
+Fortran books, and other online resources.
+
+If you haven't yet, please explore the website and [let us know](https://github.com/fortran-lang/fortran-lang.org/issues)
+if you have any suggestions for improvement.
+Specifically, we'll be focusing on the [Learn](/learn) page and its mini-books
+in the coming months.
+Please help us make them better!
+
+Here are some specific items that we worked on:
+
+* [#90](https://github.com/fortran-lang/fortran-lang.org/pull/90)
+WIP: Mini-book on building programs
+* [#83](https://github.com/fortran-lang/fortran-lang.org/pull/83)
+Improving the structure and navigation of the [Learn](/learn) pages
+* [#46](https://github.com/fortran-lang/fortran-lang.org/pull/46)
+Build website previews from pull requests
+
+## Fortran Discourse
+
+On May 4 we launched the [Fortran Discourse](https://fortran-lang.discourse.group), an online discussion board
+for anything and everything Fortran related.
+You can use it discuss the Fortran language, ask for help, announce events and/or personal projects, or just lurk
+around. 
+There are already quite a few interesting discussions going on.
+Join us!
+
+## Fortran Standard Library
+
```

Comment:
```suggestion

Recently we launched a [website](https://stdlib.fortran-lang.org) for the API documentation of the Fortran Standard Library. 
The [website](https://stdlib.fortran-lang.org) is automaticaly generated by [FORD](https://github.com/Fortran-FOSS-Programmers/ford#readme).
[Code of Conduct](https://stdlib.fortran-lang.org/page/contributing/CodeOfConduct.html), [licence](https://stdlib.fortran-lang.org/page/License.html), and [workflow](https://stdlib.fortran-lang.org/page/contributing/Workflow.html) for contributing to the Fortran Standard Library can also be found on the [website](https://stdlib.fortran-lang.org/).

```

Following [this comment](https://github.com/fortran-lang/fortran-lang.org/pull/92/files#r432571024)